src/Query.js

```diff
@@ -10,6 +10,7 @@ export default class Query {
 
         this.previous_errors = [];
         this.mysql = mysql;
+        this.connection = mysql.connection;
```

Comment:
Why do we need this?

Reply:
we dont. I used it on the first try of solving it but forgot to remove it when the final solution doesnt need it.